charts/bookkeeper/values.yaml

```diff
@@ -71,6 +71,9 @@ options:
   ## Use multiple journal and ledger directories to try exploiting more parallelism at the drive level.
   journalDirectories: "/bk/journal/j0,/bk/journal/j1,/bk/journal/j2,/bk/journal/j3"
   ledgerDirectories: "/bk/ledgers/l0,/bk/ledgers/l1,/bk/ledgers/l2,/bk/ledgers/l3"
+  ## Use journal and ledger subpath if you wish to override the default subpath name used
+  # journalSubPath: ""
+  # ledgerSubPath: ""
```

Comment:
Please add indexsubpath as well to values.yaml

Reply:
There is no `indexDirs` in the values file to go with `indexSubPath`. If one is to be added to this file, so is the other. Better not to change the defaults though.